### PR TITLE
Update installing-controlplane-on-k8s.md

### PR DESCRIPTION
**Pull request by jz543fm**
**What this PR does / why we need it**:
Updates docs how to Install Control Plane on Kubernetes for (MySQL + MINIO)

**Which issue(s) this PR fixes**:
Issue [4363](https://github.com/pipe-cd/pipecd/issues/4363)

Fixes #

**Does this PR introduce a user-facing change?**:

- **How are users affected by this change**:
- **Is this breaking change**:
- **How to migrate (if breaking change)**:

### CODE REVIEW

File: docs/content/en/docs-v0.47.x/installation/install-control-plane/installing-controlplane-on-k8s.md [hunk 1]
```diff
@@ -72,9 +74,9 @@ helm upgrade -i pipecd oci://ghcr.io/pipe-cd/chart/pipecd --version {{< blocks/l
   --set-file secret.gcsServiceAccount.data=path-to-service-account-file
 ```
 
-Currently, besides `Firestore` PipeCD supports other databases as its datastore such as `MySQL`. Also as for filestore, PipeCD supports `AWS S3` and `MINIO` either.
+#### Using MySQL and MINIO
 
-For example, in case of using `MySQL` as datastore and `MINIO` as filestore, the ControlPlane configuration will be as follow:
+For example, in case of using `MySQL` as datastore and `MINIO` as filestore, remember that url of database should be in format: `<user>:<password>@tcp(<ip>:<port>)`, the ControlPlane configuration will be as follow:
```
Comment (t-kikuc) [inline docs/content/en/docs-v0.47.x/installation/install-control-plane/installing-controlplane-on-k8s.md:79]: [nits]
> remember that url of database should be in format: `<user>:<password>@tcp(<ip>:<port>)`

How about moving it to the following yaml to clarify where this sentence mentions? 
like this:

```yaml
  datastore:
    type: MYSQL
    config:
      url: {YOUR_MYSQL_ADDRESS} # format: <user>:<password>@tcp(<ip>:<port>)

```

**Review by t-kikuc — COMMENTED** [review]
Thank you for your great improvement!
I commented a nit.
Other points seem good 👍

**Comment by jz543fm** [comment]
How to fix these commits with DCO check?

**Comment by jz543fm** [comment]
Closed due to investigating the problem with DCO check (signed-off commits)

**Pull request closed without merging**